client/components/plans/plans-compare/index.jsx

```diff
@@ -117,15 +112,15 @@ var PlansCompare = React.createClass( {
 		}
 	},
 
-	isSubmitting: function() {
+	isSubmitting() {
 		if ( ! this.props.transaction ) {
 			return false;
 		}
 
-		return this.props.transaction.step.name === transactionStepTypes.SUBMITTING_WPCOM_REQUEST
+		return this.props.transaction.step.name === SUBMITTING_WPCOM_REQUEST
```

Comment:
I think @Tug's point was about importing ambiguous path builder names (e.g. `plans()` vs. `plansPaths.plans()`). I agree with Stephane that `SUBMITTING_WPCOM_REQUEST` isn't ambiguous.